docs/ossm/cert-manager/README.md

```diff
@@ -121,107 +147,133 @@ You can install the cert-manager tool to manage the lifecycle of TLS certificate
-    ```sh
-    oc get istiorevisions
-    ```
+    - Deploy the sample `httpbin` app.
 
-    c. Add the injection label for your active revision to the `sample` namespace:
+      ```sh
+      oc apply -n sample -f https://raw.githubusercontent.com/istio/istio/refs/heads/master/samples/httpbin/httpbin.yaml
+      ```
 
-    ```sh
-    oc label namespace sample istio.io/rev=<your-active-revision-name> --overwrite=true
-    ```
+    - Deploy the sample `sleep` app.
 
-    d. Deploy the HTTP and sleep apps:
+      ```sh
+      oc apply -n sample -f https://raw.githubusercontent.com/istio/istio/refs/heads/master/samples/sleep/sleep.yaml
+      ```
 
-    ```sh
-    oc apply -n sample -f https://raw.githubusercontent.com/istio/istio/refs/heads/master/samples/httpbin/httpbin.yaml
-    oc apply -n sample -f https://raw.githubusercontent.com/istio/istio/refs/heads/master/samples/sleep/sleep.yaml
-    oc rollout status deployment httpbin sleep
-    ```
+    - Wait for both apps to become ready.
 
-    e. Verify that sleep can access the httpbin service:
+      ```sh
+      oc rollout status -n sample deployment httpbin sleep
+      ```
 
-    ```sh
-    oc exec "$(oc get pod -l app=sleep -n sample \
-      -o jsonpath={.items..metadata.name})" -c sleep -n sample -- \
-      curl http://httpbin.sample:8000/ip -s -o /dev/null \
-      -w "%{http_code}\n"
-    ```
+    - Verify that sleep can access the httpbin service:
 
-    Example output
+      ```sh
+      oc exec "$(oc get pod -l app=sleep -n sample \
+        -o jsonpath={.items..metadata.name})" -c sleep -n sample -- \
+        curl http://httpbin.sample:8000/ip -s -o /dev/null \
+        -w "%{http_code}\n"
+      ```
 
-    ```sh
-    200
-    ```
+      Example output
 
-    f. Verify `httpbin` workload certificate matches what is expected:
+      ```sh
+      200
+      ```
 
-    ```sh
-    istioctl proxy-config secret -n sample $(oc get pods -n sample -o jsonpath='{.items..metadata.name}' --selector app=httpbin) -o json | jq -r '.dynamicActiveSecrets[0].secret.tlsCertificate.certificateChain.inlineBytes' | base64 --decode | openssl x509 -text -noout
-    ```
+    - Verify `httpbin` workload certificate matches what is expected:
 
-    Example output
+      ```sh
+      istioctl proxy-config secret -n sample $(oc get pods -n sample -o jsonpath='{.items..metadata.name}' --selector app=httpbin) -o json | jq -r '.dynamicActiveSecrets[0].secret.tlsCertificate.certificateChain.inlineBytes' | base64 --decode | openssl x509 -text -noout
+      ```
 
-    ```sh
-    ...
-    Issuer: O = cert-manager + O = cluster.local, CN = istio-ca
-    ...
-    X509v3 Subject Alternative Name:
-      URI:spiffe://cluster.local/ns/sample/sa/httpbin
-    ```
+      Example output
+
+      ```sh
+      ...
+      Issuer: O = cert-manager + O = cluster.local, CN = istio-ca
+      ...
+      X509v3 Subject Alternative Name:
+        URI:spiffe://cluster.local/ns/sample/sa/httpbin
+      ```
 
 ### `RevisionBased` Upgrades
```

Comment:
What is this RevisonBased Upgrades title from?

Reply:
It's a separate section exclusively for `RevisionBased` upgrades. Maybe it should just be `Upgrades - RevisionBased`?

Reply:
Ah I see - didn't see that. I think it's ok.